ui/resizable.js

```diff
@@ -1040,14 +1028,7 @@ $.ui.plugin.add("resizable", "alsoResize", {
 					el.css(style);
 				});
 			};
-
-		if (typeof(o.alsoResize) === "object" && !o.alsoResize.nodeType) {
-			$.each(o.alsoResize, function(exp, c) {
```

Comment:
It looks like this was supporting an undocumented format of the option where the keys were selectors and the values were contexts to search in. I'm ok dropping this since I don't think this was *ever* documented, but we should be aware of it.